### ISSUE TITLE
Docker image error

### ISSUE THREAD

**Issue by AntonShevchuk** [astefanutti/decktape#284]
Hi

I try to use the newest version of the docker image and receive a very strange error:

```
Error: Unable to write GSUB: script latn has no default language system.
    at fail (/decktape/node_modules/opentype.js/dist/opentype.js:919:12)
    at Object.argument (/decktape/node_modules/opentype.js/dist/opentype.js:926:10)
    at /decktape/node_modules/opentype.js/dist/opentype.js:2032:20
    at recordList (/decktape/node_modules/opentype.js/dist/opentype.js:1990:33)
    at new ScriptList (/decktape/node_modules/opentype.js/dist/opentype.js:2029:10)
    at Object.makeGsubTable [as make] (/decktape/node_modules/opentype.js/dist/opentype.js:6874:50)
    at Object.fontToSfntTable [as fontToTable] (/decktape/node_modules/opentype.js/dist/opentype.js:7220:27)
    at Font.toTables (/decktape/node_modules/opentype.js/dist/opentype.js:13545:18)
    at Font.toArrayBuffer (/decktape/node_modules/opentype.js/dist/opentype.js:13559:27)
    at file:///decktape/decktape.js:400:73
```

With version 3.7 all was good, I just convert index.html with reveal.js:

```
node /decktape/decktape.js --chrome-path chromium-browser ./index.html ./index.pdf
```

**Comment by astefanutti** [comment]
That looks like a regression introduced by the migration to opentype.js (#280).

I haven't faced the issue on the number of presentations I've tested. Could you share your presentation, so I can try to reproduce? Or give some details that could help identify the root cause?

**Comment by AntonShevchuk** [comment]
I can share it with you personal, it is not a public project yet.
You can mail me, I will send you the files.

**Comment by eitansuez** [comment]
fwiw i also just ran into this very same error (also a private repository, sorry).

**Comment by christopher-besch** [comment]
It also fails with one of my public presentations. Try it with:
```bash
docker run --rm -t --net=host -v `pwd`:/slides astefanutti/decktape https://present.chris-besch.com/2022_06_24_reveal_example/ slides.pdf
```

**Comment by JanoschDeurer** [comment]
Same here, for all our slides. For me going back to `astefanutti/decktape:3.7` worked.

**Comment by bast** [comment]
A public example where 3.8 fails and 3.7 works: https://cicero.xyz/v3/remark/0.14.0/github.com/constructicon/presentations/main/open-code-open-data.md/

**Comment by astefanutti** [comment]
It should be fixed in [3.9.0](https://github.com/astefanutti/decktape/releases/tag/v3.9.0) release.

Thanks a lot for the report!

**Comment by bast** [comment]
Thanks a lot for the fix and this great tool!
